src/RcppExports.cpp

```diff
@@ -1,6 +1,7 @@
 // This file was generated by Rcpp::compileAttributes
 // Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
+#include "Rblpapi_types.h"
```

Comment:
What version of Rcpp do you have installed?

When I add a quick testfile `src/foo.cpp` and re-run `compileAttributes.h` I do *not* get an additional export for `Rblpapi_types.h`.  Something is different at your side, and you need to look into that.